testing/python/metafunc.py

```diff
@@ -220,6 +220,53 @@ def func(x, y): pass
         assert metafunc._calls[0].params == dict(x=1,y=2, unnamed=1)
         assert metafunc._calls[1].params == dict(x=1,y=3, unnamed=1)
 
+    def test_parametrize_indirect_list(self):
```

Comment:
Please add a docstring or in the name of the function, a reference to the issue number